### PR TITLE
Created gnome_sort

### PR DESCRIPTION
**Pull request by Budcoderr**

### CODE REVIEW

File: sort/gnome_sort [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<math.h>
+
+void main()
+{
+  int arr[]={-5,6,-7,10,4,1};
```
Comment (abiduzz420) [inline sort/gnome_sort]: Could you dynamically allocate the array ? @Budcoderr

Comment (Budcoderr) [reply sort/gnome_sort]: Shall i use linked list or just ask the user how many elements he wants to enter in the array and then allocate it?

**Comment by prateekiiest** [comment]
@Budcoderr 
see issue #82

File: sort/gnome_sort (hunk 1 above)
Comment (abiduzz420) [reply sort/gnome_sort]: Go with the array

Comment (Budcoderr) [reply sort/gnome_sort]: i have updated it with array

**Comment by soubh1k** [comment]
I am closing this pull request. The current directory structure has been changed. Fork the new repo with the new directory structure. Insert the files in the new directory structure along with the appropriate readme's. Then create a new Pull Request.

Also don't take input from the user. Create your own fixed array.

**Pull request closed without merging**